car_info cur_car
car_info past_car
peds_info cur_peds
peds_info past_peds
ped_belief[] believes
nav_msgs/Path plan
geometry_msgs/Twist action_reward